src/Components/OATPropertyEditor/Editor.tsx

```diff
@@ -49,12 +49,34 @@ const Editor = ({
     const [draggingProperty, setDraggingProperty] = useState(false);
     const enteredTemplateRef = useRef(null);
     const enteredPropertyRef = useRef(null);
+    const editorColumnRef = useRef();
     const { model, templatesActive } = state;
+    const [editorColumnBoundingRect, setEditorColumnBoundingRect] = useState(
+        null
+    );
 
     const propertiesKeyName = getModelPropertyCollectionName(
         model ? model['@type'] : ModelTypes.interface
     );
 
+    useEffect(() => {
+        if (editorColumnRef.current) {
+            setEditorColumnBoundingRect(
+                editorColumnRef.current.getBoundingClientRect()
```

Comment:
It'd be really great if we didn't need to use JS to figure out the height.

Did you have a look at the <Stack> component?

Reply:
Feel free to ping me if you want to brainstorm a bit on this one...